src/site/content/en/blog/local-fonts/index.md

```diff
@@ -179,6 +179,11 @@ if ('queryLocalFonts' in self) {
 }
 ```
 
+{% Aside %}
+The Local Font Access API is currently only available on desktop versions of Chromium, but not on
+Android.
```

Comment:
I would not mention Android. As in iOS should be mentioned as well and others.
Or maybe say "mobile" instead of Android. Your choice.